include/oboe/AudioStreamBuilder.h

```diff
@@ -399,6 +401,10 @@ class AudioStreamBuilder : public AudioStreamBase {
      */
     Result openManagedStream(ManagedStream &stream);
 
+    Result openSharedStream(std::shared_ptr<oboe::AudioStream> &sharedStream);
+
+    ResultWithValue<std::shared_ptr<oboe::AudioStream>> openSharedStream();
```

Comment:
I would advocate for changing the method name to `openStream`. 

We already have the concept of a "shared audio stream" which means `sharingMode=shared` so `openSharedStream` is likely to cause confusion with users thinking "but I want an EXCLUSIVE stream, not a SHARED stream?!" or similar.

Reply:
Good point. We still have a "shared_ptr" but I think that is less confusing. One can have a shared_ptr to an eXCLUSIVE stream. I will make these changes.